test/xunit.analyzers.tests/AssertEqualLiteralValueShouldBeFirstFixerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Xunit.Analyzers
+{
+    public class AssertEqualLiteralValueShouldBeFirstFixerTests
+    {
+        readonly DiagnosticAnalyzer analyzer = new AssertEqualLiteralValueShouldBeFirst();
+        readonly CodeFixProvider fixer = new AssertEqualLiteralValueShouldBeFirstFixer();
+
+        static readonly string Template = @"
+public class TestClass
+{{
+    [Xunit.Fact]
+    public void TestMethod()
+    {{
+        var i = 0;
+        Xunit.{0};
+    }}
+}}";
+
+        [Fact]
+        public async void SwapArguments()
+        {
+            var source = string.Format(Template, "Assert.Equal(i, 0)");
+            var expected = string.Format(Template, "Assert.Equal(0, i)");
+
+            var actual = await CodeAnalyzerHelper.GetFixedCodeAsync(analyzer, fixer, source);
+
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public async void NamedArgumentsOnlySwapsArgumentValues()
```

Comment:
How about adding a test for the case of named arguments used in the "correct" positions, e.g. `var source = string.Format(Template, "Assert.Equal(expected: i, actual: 0)");`

Reply:
Nice one. Added.